packs/feats/sea-glass-guardians.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>Eerily beautiful elemental water beings race around you, eager to protect and heal you and your allies. Their forms vary and might include eels formed of undulating water or ice crystals whirling in the shape of a jellyfish. The guardians flow around combatants and don't occupy a space. They attempt to intercept all dangers, granting you and your allies within your kinetic aura a +1 status bonus to AC and saving throws.</p>\n<p>If any creature affected is critically hit or critically fails at a saving throw against an attack, effect from an enemy, or hazard and remains above 0 HP, the guardians reach out to heal that creature. The creature regains 4d8+8 HP, and the impulse ends. If the creature is in water, the healing dice are d10s instead of d8s.</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Stance: Sea Glass Guardians]</p>\n<hr />\n<p><strong>Level (+3)</strong> The healing increases by 1d8+4.</p>\n<p>[[/r ((floor((max(12,@actor.level)-12)/3)+4)d8+(floor((max(12,@actor.level)-12)/3)*4+8))[healing]]]{Leveled HP}</p>"
+            "value": "<p>Eerily beautiful elemental water beings race around you, eager to protect and heal you and your allies. Their forms vary and might include eels formed of undulating water or ice crystals whirling in the shape of a jellyfish. The guardians flow around combatants and don't occupy a space. They attempt to intercept all dangers, granting you and your allies within your kinetic aura a +1 status bonus to AC and saving throws.</p>\n<p>If any creature affected is critically hit or critically fails at a saving throw against an attack, effect from an enemy, or hazard and remains above 0 HP, the guardians reach out to heal that creature. The creature regains 4d8+8 HP, and the impulse ends. If the creature is in water, the healing dice are d10s instead of d8s.</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Stance: Sea Glass Guardians]</p><hr /><p><strong>Level (+3)</strong> The healing increases by 1d8+4.</p>\n<p>@Damage[((floor((max(12,@actor.level)-12)/3)+4)d8+(floor((max(12,@actor.level)-12)/3)*4+8))[healing]]{Leveled HP}</p>"
```

Comment:
```suggestion
            "value": "<p>Eerily beautiful elemental water beings race around you, eager to protect and heal you and your allies. Their forms vary and might include eels formed of undulating water or ice crystals whirling in the shape of a jellyfish. The guardians flow around combatants and don't occupy a space. They attempt to intercept all dangers, granting you and your allies within your kinetic aura a +1 status bonus to AC and saving throws.</p>\n<p>If any creature affected is critically hit or critically fails at a saving throw against an attack, effect from an enemy, or hazard and remains above 0 HP, the guardians reach out to heal that creature. The creature regains @Damage[((floor((max(12,@actor.level)-12)/3)+4)d8+(floor((max(12,@actor.level)-12)/3)*4+8))[healing]] HP, and the impulse ends. If the creature is in water, the healing dice are d10s instead of d8s.</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Stance: Sea Glass Guardians]</p><hr /><p><strong>Level (+3)</strong> The healing increases by 1d8+4.</p>"
```